src/App.js

```diff
@@ -1,26 +1,37 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import indigo from '@material-ui/core/colors/indigo';
+import deepOrange from '@material-ui/core/colors/deepOrange';
+
+import Login from './pages/Login';
+import Home from './pages/Home';
+
+const theme = createMuiTheme({
+  palette: {
```

Comment:
Consider moving theme setup to a seperate file (e. g. : themeConfig.js), other configurations should be moved to config files so we avoid spaghetti code